src/main/java/com/mapzen/pelias/PeliasMapPositionListener.java

```diff
@@ -0,0 +1,6 @@
+package com.mapzen.pelias;
+
+public interface PeliasMapPositionListener {
```

Comment:
What do you think about re-using the `PeliasLocationProvider` interface instead of adding a new one?